synapse/config/metrics.py

```diff
@@ -58,6 +76,16 @@ def generate_config_section(self, report_stats=None, **kwargs):
         #sentry:
         #    dsn: "..."
 
+        # Flags to enable metrics which are not suitable to be enabled by
+        # default, either for performance reasons or limited use.
+        #
+        # known_servers: Publish synapse_federation_known_servers, a gauge of
```

Comment:
our style doc says to uncomment `metrics_flags` and comment each subentry next to it, rather than at the top of the block (for which you need to make it cope with `metrics_flags` being `None`).

Reply:
done